encodings/fastlanes/src/bitpacking/compute/filter.rs

```diff
@@ -110,6 +104,36 @@ fn filter_indices<T: NativePType + BitPacking + ArrowNativeType>(
     values
 }
 
+pub fn chunked_indices<F: FnMut(usize, &[usize])>(
```

Comment:
does this need to be truly public? can we just have it be accessible within the module?

Reply:
Pub(crate) is enough